CommonClient.py

```diff
@@ -942,11 +944,10 @@ async def process_server_cmd(ctx: CommonContext, args: dict):
             logger.info('--------------------------------')
             logger.info('Room Information:')
             logger.info('--------------------------------')
-            version = args["version"]
-            ctx.server_version = Version(*version)
+            ctx.server_version = Version.from_network_dict(args["version"])
```

Comment:
I think this is maybe backwards. What would be better is iterating over the decoded object and upcasting it. (I.e. writing our own deserializer hook system on top of orjson)